### DISCUSSION TITLE
Pinned Tab Ordering

### DISCUSSION THREAD

**Discussion by ldelossa** [qutebrowser/qutebrowser#6540] [Q&A] [answered]
Hello,

Is there a way to have any new tabs opened from a pinned tab placed after *all* pinned tabs. 

Right now if I have pinned tabs [1(pinned),2(pinned)] and open a link in a new tab from 1 we wind up with [1(pinned), 2(normal, new tab), 3(pinned)]. I'd like to end up with [1(pinned),2(pinned),3(normal, new tab)]. Where new tab is the link that was opened in a new tab from pinned tab 1.

**Comment by The-Compiler** [comment, marked as answer]
See #2898
